crates/volta-core/src/tool/package/mod.rs

```diff
@@ -3,14 +3,17 @@ use std::path::{Path, PathBuf};
 
 use super::registry::PackageDetails;
 use super::{check_fetched, debug_already_fetched, info_fetched, FetchStatus, Tool};
-use crate::error::{Context, ErrorKind, Fallible};
 use crate::fs::{dir_entry_match, remove_dir_if_exists, remove_file_if_exists};
 use crate::inventory::package_available;
 use crate::layout::volta_home;
 use crate::session::Session;
 use crate::shim;
 use crate::style::{success_prefix, tool_version};
 use crate::sync::VoltaLock;
+use crate::{
+    error::{Context, ErrorKind, Fallible},
+    fs::ok_if_not_found,
+};
```

Comment:
To keep the diffs simpler, we've generally tried to keep our use statements to single modules at a time. So instead of grouping the submodules together like this, we would have 2 separate lines:

```rust
use crate::error::{Context, ErrorKind, Fallible};
use crate::fs::ok_if_not_found;
```
(With the appropriate ordering provided by `cargo fmt`.